### PR TITLE
(core) Remove bool filtered rows immediately on change

### PR DESCRIPTION
**Pull request by wunter8**
See #514

### CODE REVIEW

File: app/client/components/BaseView.js [hunk 1]
```diff
@@ -581,7 +581,8 @@ BaseView.prototype._saveEditRowField = function(editRowModel, colName, value) {
       // Display this rowId, even if it doesn't match the filter
       .then((result) => {
         if (!this.isDisposed()) {
-          this._sectionFilter.addTemporaryRow(rowId);
+          const colType = this.currentColumn().pureType();
+          this._sectionFilter.addTemporaryRow(rowId, colType);
```
Comment (georgegevoian) [inline app/client/components/BaseView.js:585]: Other code also calls `SectionFilter.addTemporaryRow`, so I think we'll want to do the check outside of it. I'd suggest doing this instead:

```
if (!this.isDisposed() && this.currentColumn().pureType() !== 'Bool') {
  this._sectionFilter.addTemporaryRow(rowId);
}
```

**Review by georgegevoian — CHANGES_REQUESTED** [review]
Thanks for opening the PR @wunter8.

File: app/client/components/BaseView.js (hunk 1 above)
Comment (wunter8) [reply app/client/components/BaseView.js]: Sounds good :+1:

**Review by georgegevoian — APPROVED** [review]
Thanks @wunter8!

**Pull request merged**